SITE_CONTRIBUTION.md

```diff
@@ -1,166 +1,130 @@
-
-**Linking to a section within another page:**
-
-Unfortunately, the `{% link %}` templating method does not allow linking to a section within a page. Use this format when linking to a section within another page.
-`{% <Path to page post generation/<section> %}` or `/docs/v1.0.0/deploy-options/#Host-Networking`
-
-A key difference between using `{% link %}` and the above method is that `{% link %}` will resolve to pages final location. To use the above method, you need to point the link to the location where the document will end up. For example, files in the `_guides` directory will be located in `/guides` on the live site. Therefore, the link for `_guides/cert-manager` is `/guides/cert-manager`.
-
-Additionally, the above method will not follow any permalink changes or page-specific permalink.
-
-**Linking to the project contour GitHub repo:**
-
-The GitHub metadata plugin populates Jekyll variables with data from GitHub during build. When inserting a link to the Project Contour repository, use the prefix `{{< param github_url >}}`.
-
-A full list of variables can be found in the [jekyll-github-metadata documentation][4].
-
-**Linking to an external page:**
-
-Links to external pages are done using the full URL, `[1]: https://projectcontour.io` for example.
-
-## Proofread and spell check
-
-If you're making any content submissions, please proofread and spell check your work.
-
-Spell check can be run using `make check`.
-
-## Local Site Testing
-
-Perform local tests before creating a PR to detect issues such as broken links.
-
-**Using docker:**
+## Linking to resources
+Reference table links are preferred over inline links as reference tables make it easier to find and update links.
+Reference table links use the following Markdown format:
+```
+If you encounter issues, review the [troubleshooting][17] page, [file an issue][6], or talk to us on the [#contour channel][12] on Kubernetes Slack.
+```
 
-The easiest way to run the site locally is by using the top-level [Makefile](./Makefile). This has a `site-devel` target and runs a [Jekyll][6] development server in Docker, which is useful for checking local documentation changes.
-By default, the site will be available on [port 4000](http://127.0.0.1:4000/).
+A reference table, located at the end of the Markdown file, uses the following format:
+```
+[1]: /docs/{{< param latest_version >}}/deploy-options
+[2]: /CONTRIBUTING.md#contribution-workflow
+[3]: {{< param github_url >}}/issues
+[4]: https://httpbin.org/
+[5]: https://github.com/projectcontour/community/wiki/Office-Hours
+[6]: {{< param slack_url >}}
+[7]: https://github.com/bitnami/charts/tree/master/bitnami/contour
+[8]: https://www.youtube.com/watch?v=xUJbTnN3Dmw
+```
 
-Note that the version of [Jekyll][6] used by the site is pinned in the [Gemfile](./site/Gemfile.lock). The `site-devel` target uses the matching versioned Docker image to ensure compatibility.
+## Using URL parameters
```

Comment:
I know the section just above shows examples of how to use these params, but I would include an example of how to use them in this section ("Using URL parameters") as well.

Reply:
added a few sample reference links.